documentation/modules/configuring/con-scaling-kafka-clusters.adoc

```diff
@@ -3,32 +3,44 @@
-
-[discrete]
-== Broker addition
-
-The primary way of increasing throughput for a topic is to increase the number of partitions for that topic.
-That works because the extra partitions allow the load of the topic to be shared between the different brokers in the cluster.
-However, in situations where every broker is constrained by a particular resource (typically I/O) using more partitions will not result in increased throughput.
-Instead, you need to add brokers to the cluster.
-
-When you add an extra broker to the cluster, Kafka does not assign any partitions to it automatically.
-You must decide which partitions to reassign from the existing brokers to the new broker.
-
-Once the partitions have been redistributed between all the brokers, the resource utilization of each broker is reduced.
-
-[discrete]
-== Broker removal
-
-If you are using `StatefulSets` to manage broker pods, you cannot remove _any_ pod from the cluster.
-You can only remove one or more of the highest numbered pods from the cluster.
-For example, in a cluster of 12 brokers the pods are named `_cluster-name_-kafka-0` up to `_cluster-name_-kafka-11`.
-If you decide to scale down by one broker, the `_cluster-name_-kafka-11` will be removed.
-
-Before you remove a broker from a cluster, ensure that it is not assigned to any partitions.
-You should also decide which of the remaining brokers will be responsible for each of the partitions on the broker being decommissioned.
-Once the broker has no assigned partitions, you can scale the cluster down safely.
+Scaling Kafka clusters by adding brokers can increase the performance and reliability of the cluster. 
+Adding more brokers increases available resources, allowing the cluster to handle larger workloads and process more messages. 
+It can also improve fault tolerance by providing more replicas and backups. 
+Conversely, removing underutilized brokers can reduce resource consumption and improve efficiency. 
+Scaling must be done carefully to avoid disruption or data loss.
+By redistributing partitions across all brokers in the cluster, the resource utilization of each broker is reduced, which can increase the overall throughput of the cluster.
+
+NOTE: To increase the throughput of a Kafka topic, you can increase the number of partitions for that topic. 
+This allows the load of the topic to be shared between different brokers in the cluster. 
+However, if every broker is constrained by a specific resource (such as I/O), adding more partitions will not increase the throughput. 
+In this case, you need to add more brokers to the cluster.
+
+Change the `Kafka.spec.kafka.replicas` configuration to add or reduce the number of brokers.
+In the following example, the number of brokers is set at 3.
+A replica factor of 3 in Kafka means that each partition of a topic is replicated across three brokers. 
+This provides fault tolerance and ensures that if one of the brokers fails, other brokers are still available.
```

Comment:
This is a bit confusing I think. Strictly speaking, it is correct. But the `Kafka.spec.kafka.replicas` does not have any direct impact on the actual replication factor. It is only a prerequisite (e.g. to use replication factor 3, you need at least 3 replicas).

I think we should remove this here or clarify this.

Reply:
Assuming I understood what we want to cover, I've tried to clarify as follows:

Adjusting the `Kafka.spec.kafka.replicas` configuration affects the number of brokers in the cluster that act as replicas. 
The actual replication factor for topics is determined by settings for the `default.replication.factor` and `min.insync.replicas`, and the number of available brokers. 
For example, a replication factor of 3 means that each partition of a topic will be replicated across three brokers, ensuring fault tolerance in the event of a broker failure.